backend/src/main/java/com/redhat/cloud/notifications/routers/EventResource.java

```diff
@@ -130,4 +133,48 @@ public Page<EventLogEntry> getEvents(@Context SecurityContext securityContext, @
         page.setLinks(links);
         return page;
     }
+
+    static EventLogEntryActionStatus fromNotificationStatus(NotificationStatus status) {
+        switch (status) {
+            case SENT:
+                return EventLogEntryActionStatus.SENT;
+            case SUCCESS:
+                return EventLogEntryActionStatus.SUCCESS;
+            case PROCESSING:
+                return EventLogEntryActionStatus.PROCESSING;
+            case FAILED_EXTERNAL:
+            case FAILED_INTERNAL:
+                return EventLogEntryActionStatus.FAILED;
+            default:
+                Log.warnf("Uncovered status found:[%s]. This is a bug", status);
+                return EventLogEntryActionStatus.UNKNOWN;
+        }
+    }
+
+    static Set<NotificationStatus> toNotificationStatus(Set<EventLogEntryActionStatus> statusSet) {
```

Comment:
We could use a test for each switch case.

Reply:
aaand done.